### DISCUSSION TITLE
Way to see step by step output of RAG pipeline

### DISCUSSION THREAD

**Discussion by prateekkohli21** [infiniflow/ragflow#3191] [Q&A]
I want to see output of each step of RAG pipeline. Is there any way I can see these? Or maybe enable debug logging fir this?

**Comment by cyhasuka** [comment]
First change the `LoggerFactory.LEVEL` in all `settings.py` to `10`
Then just run `tail -f logs/api/*.log`.
Or if you run ragflow in docker, Enter the container first.
